iina/AppData.swift

```diff
@@ -11,7 +11,7 @@ import Cocoa
 struct AppData {
   
   /** time interval to sync play pos */
-  static let getTimeInterval: Double = 0.5
+  static let getTimeInterval: Double = 0.1
```

Comment:
I assumed getting a number from mpv and asking the slider to refresh would be pretty lightweight. I performed a profiling to validate that. And it showed that `MainWindowController.updatePlayTime()` weighted 0.1% in terms of CPU time, and `PlayCore.syncUITime()` weighted 0.0%. So I guess the performance has not been affected.

The commits have been rebased to the tip of develop branch to resolve the conflicts caused by the "remove trailing spaces" commit.